backend/db_context/tables.py

```diff
@@ -17,20 +17,25 @@ class ReactChatDocument(Document):
     created_at = DateTimeField(default=datetime.utcnow, index=True)
 
     def __init__(self, *args, **values):
-        values = {underscore(k): v for k, v in values}
+        values = {underscore(k): v for k, v in values.items()}
         super().__init__(*args, **values)
 
     def save(self, *args, **kwargs):
-        self.updated_on = datetime.utcnow()
+        self.updated_at = datetime.utcnow()
         return super().save(*args, **kwargs)
 
+    def update(self, **kwargs):
+        kwargs = {underscore(k): v for k, v in kwargs.items()}
```

Comment:
change camelcase to underscore so we can create model directly from the client data